tensorrt_llm/_torch/auto_deploy/transform/library/fuse_rope_mla.py

```
@@ -0,0 +1,497 @@
+
+
+def _compute_rotary_cos_sin_from_config(
+    factory: ModelFactory,
+) -> Optional[torch.Tensor]:
+    """Compute rotary_cos_sin from the HuggingFace model config.
+
+    This is the fallback when buffer tracing fails.
+    """
+    try:
+        model_config, _ = factory._get_model_config()
+    except Exception:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)